include/rdma/fabric.h

```diff
@@ -49,7 +49,7 @@ extern "C" {
 
 enum {
 	FI_MAJOR_VERSION	= 1,
-	FI_MINOR_VERSION	= 0,
+	FI_MINOR_VERSION	= 1,
```

Comment:
Because this breaks apps, I _think_ this change as proposed would result in bumping the major version.  I'll check the versioning documentation to be sure.